### ISSUE TITLE
[BUG] Frame AppLandingPage is not loaded - Exception When Opening App

### ISSUE THREAD

**Issue by ryanpiran** [Microsoft/EasyRepro#862]
## Bug Report

**EasyRepro Version**
  - [X] Microsoft Dynamics 365 Online Version 9.1 (9.1.x) (DB 9.1.x) online
  
**UCI or Classic Web**
  - [X] UCI (XrmApp)

**Online or On Premise**
  - [X] Online

**Browser**
  - [X] Chrome

**Describe the bug**  
Upon calling Xrm.App.Navigation.OpenApp(); the following exception is thrown:
System.InvalidOperationException: 'Frame AppLandingPage is not loaded. Exception: No frame element found with name or id AppLandingPage'. Preventing any of our U.I tests to run any further. This issue was introduced when updating our EasyRepro version from 9.1.0.11711 to 9.1.0.12013. One thing to note is that we have recently turned off the Default App within Dynamics 365 and are now only using the UCI. 

```csharp
xrmApp.OnlineLogin.Login(_xrmUri, _username, _password);

xrmApp.Navigation.OpenApp("Sales Hub");
```
 -->

**Comment by TYLEROL** [comment]
@ryanpiran - We merged the latest code from develop into releases/v9.1 and published a new 9.1.0.17138 NuGet package today. Could you please try this against the latest branch? I tried earlier today but did not see any issues.

Happy to re-evaluate if you're still seeing issues.

-Tyler

**Comment by swethagampala** [comment]
Hello. I cloned latest v9.1 version. I still see this error:
Message: Test method Microsoft.Dynamics365.UIAutomation.Sample.Web.CreateAccount.WEBTestCreateNewAccount threw exception:
System.InvalidOperationException: Login page failed
I am using IE. It logs in with username, password and IE ask to remember dialog and then test fails.
I added this line also:

xrmBrowser.LoginPage.Login(_xrmUri, _username, _password);
xrmBrowser.GuidedHelp.CloseGuidedHelp();
xrmBrowser.Dialogs.CloseWarningDialog();
xrmBrowser.Navigation.OpenApp("Sales Hub");
xrmBrowser.ThinkTime(500);
xrmBrowser.Navigation.OpenSubArea("Sales", "Accounts");

Please help. I am on version 9.1.x sandbox.
